CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 ## [Unreleased]
 
 ### Changed
+- [#970](https://github.com/JabRef/jabref/issues/970): Implementation of shared database support (base system)
```

Comment:
I think the description could be more expressive, maybe along the lines "automatic synchronization with external database (MySQL, ..)"